lib/redis_cluster/pool.rb

```diff
@@ -54,8 +54,30 @@ def pipelined(args, &block)
       random_node.execute :pipelined, args, &block
     end
 
+    # Implements scan across all nodes in the pool.  Encodes the current node in the returned cursor.
+    # Cursors will behave strangely if the node list changes during iteration.
+    def scan(args)
+      cursor = args.first
+      options = args[1] || {}
+      orig_cursor, node_index = decode_scan_cursor(cursor)
+      next_cursor, result = @nodes[node_index].execute("scan", [orig_cursor.to_s, options])
+      [ encode_scan_cursor(next_cursor.to_i, node_index), result ]
```

Comment:
No need for spaces inside the `[ ]` right?

Reply:
Yes, thanks...